### ISSUE TITLE
[QA] Flickering Trees When Facing North

### ISSUE THREAD

**Issue by Ludmilafantaniella** [decentraland/unity-explorer#2240]
### **Explorer Alpha build versions:**

- **v0.29.0**

### **Issue Description:**
When facing north at coordinates (50, 110), trees flicker as the camera is moved.

### **LOGS:**
[Flickering issue.zip](https://github.com/user-attachments/files/17196507/Flickering.issue.zip)

### **STR:** 

1. Launch the explorer.
2. Use the command `/goto 50,110`.
3. Observe the trees while moving the camera.

### ❌ Actual Result
Trees flicker when the camera is moved while facing north.

https://github.com/user-attachments/assets/32acfe1b-c79c-494f-9f2a-f07d20853f2c

### ✅ Expected Result
Trees should render smoothly without flickering when the camera is moved.

#### **Additional Notes:**
Windows 11. 12th Gen Intel(R) Core(TM) i7-12650H 2.30 GHz. nvidia geforce RTX
Macbook M1 Pro Sonoma 14.5

**Comment by GBirch33** [comment]
it would seem to be this scene:
![image](https://github.com/user-attachments/assets/a916597e-22e5-44a9-8071-00bc7038e0de)

**Comment by GBirch33** [comment]
More specifically this asset:
![image](https://github.com/user-attachments/assets/3d21e4de-f871-4b5d-99d2-fd5905f7edf5)

**Comment by anicalbano** [comment]
If it's scene related, should we keep the ticket open? cc @aixaCode @m3taphysics ?

**Comment by m3taphysics** [comment]
Yes we may be able to work around any odd values that is causing the problem

**Comment by m3taphysics** [comment]
Closing as duplicate
